fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -2047,51 +2064,41 @@ private void httpStreamPutImpl(TStreamLoadPutRequest request, TStreamLoadPutResu
             checkPasswordAndPrivs(cluster, request.getUser(), request.getPasswd(), request.getDb(), request.getTbl(),
                     request.getUserIp(), PrivPredicate.LOAD);
         }
+        ctx.setThreadLocalInfo();
         ctx.setEnv(Env.getCurrentEnv());
         ctx.setQueryId(request.getLoadId());
         ctx.setCluster(SystemInfoService.DEFAULT_CLUSTER);
         ctx.setCurrentUserIdentity(UserIdentity.ROOT);
```

Comment:
why use root???
Should use the actual username because we need check if current user has the related privilege!!!